client/src/components/generic/Dropdown/Dropdown.tsx

```diff
@@ -1,34 +1,30 @@
 import React from "react"
-import DownArrow from "assets/icons/downarrow.svg"
 
-type DropdownProps = React.SelectHTMLAttributes<HTMLSelectElement>
+type DropdownProps = React.SelectHTMLAttributes<HTMLSelectElement> & {
+  label?: string
```

Comment:
same here, `label`, `value` are native to `React.SelectHTMLAttributes<HTMLSelectElement>`